### PR TITLE
Feature/extend frontend with ward level boundaries

### PR DESCRIPTION
**Pull request by musamusa**
please refer to #141 for details

### CODE REVIEW

**Comment by musamusa** [comment]
@norkans7 kindly review

**Comment by ribalba** [comment]
Hello @norkans7 I am Didi the CTO of eHA and I just wanted to ask why this is taking quite a lot of time? Is there anything we can do to get this merged faster as we seem to be running behind the master tree?

**Comment by norkans7** [comment]
Hey @ribalba @musamusa , this one is not complete since it needs to use the data from the db objects instead of the old direct fetch from rapidpro, I think it can check for #133 or wait for it to be merged to see clearly the approach needed and add support for the ward level

**Comment by musamusa** [comment]
@norkans7 can you review my update while add more test?

File: ureport/utils/__init__.py [hunk 1]
```diff
@@ -557,15 +561,15 @@ def get_segment_org_boundaries(org, segment):
...
         return location_boundaries
 
-    if segment.get('location') == 'District':
-        state_id = segment.get('parent', None)
-        if state_id:
-            location_boundaries = org.boundaries.filter(level=2, parent__osm_id=state_id).values('osm_id', 'name').order_by('osm_id')
+    if segment.get('location') in ('District', 'Ward'):
+        osm_id = segment.get('parent', None)
+        if osm_id:
+            location_boundaries = org.boundaries.filter(parent__osm_id=osm_id).values('osm_id', 'name').order_by('osm_id')
```
Comment (norkans7) [inline ureport/utils/__init__.py]: I think it is useful to be explicit about the level we are looking

**Comment by musamusa** [comment]
@norkans7 looks like the last updates to master reverted a lot of stuff on my end.

**Comment by norkans7** [comment]
Right, I reverted a PR, since we need to push some codes to do a backfill. I'll notify you when to merge master,

**Comment by ribalba** [comment]
@norkans7 please be aware that we are doing this and are being paid to do so. We can not continue to adapt to changes to master as we didn't budget for this. I hope we can resolve this fairly soon otherwise we will have to adapt our budget or are forced to stop contributing.

**Comment by norkans7** [comment]
@ribalba @musamusa both this PR and #154 will be merged once the backfill of data form RapidPro is finished and we have the data to use, these past changes to master have been to optimize the way we sync the data. We currently have really big orgs and they are still growing we need to make sure this sync uses a few resources as possible(this is taking time and we are getting smarter about our approaches). And to save you for constant adaptation I can notify you once you need to update the PR so we can merge it. I am open to your suggestions as well how to approach this without breaking. Thanks for your undestanding

**Comment by norkans7** [comment]
@ribalba @musamusa consider this done, we'll take care of the conflicts at the merge time

**Pull request merged**